test/Microsoft.AspNet.Authentication.Test/Cookies/CookieMiddlewareTests.cs

```diff
@@ -748,12 +749,14 @@ private Task SignOutAsWrong(HttpContext context)
         [Fact]
         public async Task MapWillNotAffectChallenge()
         {
-            var server = TestServer.Create(app =>
+            var builder = new WebApplicationBuilder()
```

Comment:
Understood. And why not just providing a single `TestServer.Create` method taking an `Action<WebApplicationBuilder>` parameter?

```
var server = TestServer.Create(builder => {
    builder.Configure(app => { ... });
    builder.ConfigureServices(services => { ... });
});